t/plugin/zipkin.t

```diff
@@ -467,3 +467,13 @@ GET /t
 --- response_body
 1
 0
+
+
+
+=== TEST 23: no error in log phase while b3 header invalid
+--- request
+GET /echo
+--- more_headers
+b3: abc
+--- no_error_log
+[error]
```

Comment:
u can insert a new line at here to help the ci works

Reply:
ok, thk

Reply:
@Vacant2333 
my new test cast as following:

```
=== TEST 23: no error in log phase while b3 header invalid
--- request
GET /echo
--- more_headers
b3: 80f198ee56343ba864fe8b2a57d3eff7
--- response_headers
x-b3-sampled:
--- error_code: 400
--- grep_error_log eval
qr/attempt to index local 'opentracing' \(a nil value\)
--- grep_error_log_out
```

i want to assert that there is no corresponding error in the log, what should I do?

Reply:
u may need to try this https://apisix.apache.org/docs/apisix/internal/testing-framework/#executing-shell-commands, im not sure if it will works🤣, i didnt find some testcase need to do like u want

Reply:
> > i want to assert that there is no corresponding error in the log, what should I do?
> 
> You can use `no_error_log` https://github.com/apache/apisix/blob/master/docs/en/latest/plugin-develop.md?plain=1#L549

there is a error in rewrite phase, but no error in log phase.